src/pymor/operators/numpy.py

```diff
@@ -11,11 +11,14 @@
   which assemble into a |NumpyMatrixOperator|.
 - |NumpyGenericOperator| wraps an arbitrary Python function between
   |NumPy arrays| as an |Operator|.
+- |NumpyHankelOperator| implicitly constructs a Hankel operator from a |NumPy array| of Markov
```

Comment:
```suggestion
- :class:`NumpyHankelOperator` implicitly constructs a Hankel operator from a |NumPy array| of Markov
```
should make the docs build go green ..